implementations/rust/ockam/ockam_command/Cargo.toml

```diff
@@ -55,6 +55,7 @@ path = "src/bin/ockam.rs"
 anyhow = "1"
 async-recursion = { version = "1.0.0" }
 async-trait = "0.1"
+chrono = "0.4.26"
```

Comment:
Is there something from `chrono` that can't be done with `time`? `chrono` has been unmantained for a long time up until recently, and it has lots of open PR's and issues.